cmake/onnxruntime_providers.cmake

```diff
@@ -200,6 +204,39 @@ if (onnxruntime_USE_NGRAPH)
     target_link_options(onnxruntime_providers_ngraph PRIVATE "LINKER:-z, noexecstack " "LINKER:-z relro" "LINKER:-z now" "LINKER:-pie")
   endif()
 endif()
+if (onnxruntime_USE_OPENVINO)
+  file(GLOB_RECURSE onnxruntime_providers_openvino_cc_srcs
+    "${ONNXRUNTIME_ROOT}/core/providers/openvino/*.h"
+    "${ONNXRUNTIME_ROOT}/core/providers/openvino/*.cc"
+  )
+  file(GLOB_RECURSE onnxruntime_providers_openvino_py_srcs
+    "${ONNXRUNTIME_ROOT}/core/providers/openvino/openvino_mo/*.py"
+  )
+
+# Below variables point to directories within the OpenVINO installation directory
+# whose value is set in INTEL_CVSDK_DIR variable by running the setupvars.sh script
+
+if (onnxruntime_USE_OPENVINO_BINARY)
+  set(OPENVINO_INCLUDE_DIR $ENV{INTEL_CVSDK_DIR}/deployment_tools/inference_engine/include)
+  set(OPENVINO_LIB_DIR $ENV{INTEL_CVSDK_DIR}/deployment_tools/inference_engine/lib/ubuntu_16.04/intel64/)
+endif()
+
+find_package(PythonLibs REQUIRED)
```

Comment:
why does openvino has dependency on python libs? 
onnxruntime can be built without python bindings.
is it needed even if you build C++ only?

Reply:
I see now that it is leveraging python module for the MO/conversion.
This doesn't seem ideal.

Reply:
we were under the impression based on previous meetings/conversations that the provider would leverage the new NN builder to build the network at runtime without the IR.

Reply:
Hey George,

The python module that is used is only during the initialization phase and it doesn't affect inference performance. We can discuss more about this in a call later.